qiita_db/processing_job.py

```diff
@@ -219,6 +286,95 @@ def status(self, value):
             qdb.sql_connection.TRN.add(sql, [new_status, self.id])
             qdb.sql_connection.TRN.execute()
 
+    def _generate_cmd(self):
+        """Generates the command to submit the job
+
+        Returns
+        -------
+        str
+            The command to use to submit the job
+        """
+        job_dir = join(qdb.util.get_work_base_dir(), self.id)
+        software = self.command.software
+        plugin_start_script = software.start_script
+        plugin_env_script = software.environment_script
+        cmd = '%s "%s" "%s" "%s" "%s" "%s"' % (
+            qiita_config.plugin_launcher, plugin_env_script,
+            plugin_start_script, qiita_config.base_url, self.id, job_dir)
+        return cmd
+
+    def submit(self):
```

Comment:
Can't add - it will mean to run split libraries. Added a comment in the test though, for future reference.